website/docs/configuration.md

```diff
@@ -38,6 +38,25 @@ let g:neovide_transparency=0.8
 Setting `g:neovide_transparency` to a value between 0.0 and 1.0 will set the opacity of the window
 to that value.
 
+#### Background Color (Currently macOS only)
+
+```vim
+" Set transparency and background color (title bar color)
+let g:neovide_transparency=0.0
+let g:neovide_transparency_point=0.8
+let g:neovide_background_color = '#0f1117'.printf('%x', float2nr(255 * g:neovide_transparency_point))
+
+" Add keybinds to change transparency
+function! ChangeTransparency(delta)
+  let g:neovide_transparency_point = g:neovide_transparency_point + a:delta
+  let g:neovide_background_color = '#0f1117'.printf('%x', float2nr(255 * g:neovide_transparency_point))
+endfunction
+noremap <expr><D-]> ChangeTransparency(0.01)
+noremap <expr><D-[> ChangeTransparency(-0.01)
+```
```

Comment:
While it's sure nice to have a whole ready-to-use machine for changing the transparency, this page is more or less only intended to show the name of those config values, what they do and their defaults. The mechanism allowing to change this through a keybinding might have a better home in `website/docs/faq.md`- while we should probably call that thing more like a "tips & tricks" section instead.

Reply:
I separated them. Thank you!